hr_period/__manifest__.py

```diff
@@ -14,6 +14,7 @@
     'website': 'https://github.com/OCA/hr',
     'depends': [
         'hr_payroll',
+        'hr_payroll_account',
```

Comment:
Why are you adding this dependency?

Reply:
Because of the method `on_change_contract` in `'hr_payslip.py'` it calls super and super is in `hr_payroll_account`

Reply:
Then this is not the solution. Just use other name for the onchange_method and don't call super. It should be `_onchange_contract_hr_period`.

Reply:
This is still pending

Reply:
Yes. I will do that asap.